job/server/src/main/java/alluxio/master/job/metrics/DistributedCmdMetrics.java

```diff
@@ -255,7 +255,7 @@ public static void batchIncrementForCompleteStatusWithRetry(
               LOG.warn("Retry getStatus for URI {} for {}-th time, {}",
                     finalPathMapKey, retryPolicy.getAttemptCount(),
                       Arrays.toString(e.getStackTrace()));
-            } catch (NullPointerException e) {
+            } catch (RuntimeException e) {
```

Comment:
why expand the exception type? what are we including additionally ?

Reply:
Expand to catch runtime exceptions including failure to create URI for nulls, etc.